examples/movie_view_ratings.py

```diff
@@ -95,11 +96,12 @@ def process(self, line):
 
 def compute_on_beam():
     runner = fn_api_runner.FnApiRunner()  # local runner
+    public_partitions=[int(partition) for partition in FLAGS.public_partitions] if FLAGS.public_partitions is not None else None
```

Comment:
There is no need to split it, because we are using DEFINE_list, that returns list from a comma separated list. ([ref.](https://abseil.io/docs/python/guides/flags#flag-types))

Reply:
Ah, cool!